queries/toml/highlights.scm

```diff
@@ -0,0 +1,10 @@
+(bare_key) @type.builtin
+
+(pair
+  (bare_key) @property)
+
+(string) @string
+(boolean) @constant.builtin
+(integer) @number
```

Comment:
Please distinguish floats -> Contributing.md

Reply:
Thank you for your advice!

Differentiated between floating point numbers and integers.